Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/Queries/Ride.hs

```diff
@@ -93,6 +94,9 @@ findAllRidesByDriverId ::
   m [Ride]
 findAllRidesByDriverId (Id driverId) = findAllWithKV [Se.Is BeamR.driverId $ Se.Eq driverId]
 
+findSuccRideByGoHomeRequestId :: (L.MonadFlow m, Log m) => Id DDGR.DriverGoHomeRequest -> m (Maybe Ride)
+findSuccRideByGoHomeRequestId (Id ghrId) = findOneWithKV [Se.And [Se.Is BeamR.driverGoHomeRequestId $ Se.Eq (Just ghrId), Se.Is BeamR.status $ Se.Eq DRide.COMPLETED]] -- To Be added
```

Comment:
`findCompletedRideByGHRId`

Reply:
This is also a secondary key, what are the implication for it in KV @enigma-commits @vijaygupta18 @arjuna92

Reply:
This (driverGoHomeRequestId )should be added as a secondary key in the Beam file @hemantmangla